src/style.css

```diff
@@ -1,15 +1,8 @@
-fieldset {
-  margin-bottom: 20px;
-}
-
-input {
+.field_wrapper {
   margin-bottom: 10px;
 }
 
-textarea {
-  margin-bottom: 10px;
-}
-
-label {
-  margin-top: 10px;
+.box_style {
```

Comment:
following the BEM naming convention this name should have double underscore mark `box__style`